packages/salesforcedx-vscode-lwc/snippets/lwc.json

```diff
@@ -63,16 +63,16 @@
     "body": [
       "@api recordId;",
       "",
-      "@wire(getRecords, { records:[ { recordIds: [ ], fields: [ ] } ] } )",
-      "${2:records-property-name}"
+      "@wire(getRecords, { records:[ { recordIds: [ $2 ], fields: [ $0 ] } ] } )",
+      "${1:records-property-name}"
```

Comment:
I updated the tab indexes for these two snippets to be in line with the getRecord snippits.